dist/build_source_package.sh

```diff
@@ -109,6 +109,37 @@ build_pkgbuild() {
     cp -v "${temp_dir}"/pkgbuild/* "${output_dir}"/pkgbuild/
 }
 
+# Build deb source package for Debian/Ubuntu
+build_debsrc() {
```

Comment:
this is super pedantic, sorry, but we're creating a dsc which can be used to create a deb.

Reply:
Good call. Fixed!